packages/core/src/colorize.ts

```diff
@@ -0,0 +1,17 @@
+import * as colorette from 'colorette';
```

Comment:
Can we safely import `colorette` in browser?

Reply:
Unfortunately no. It requires `Node >= 10` and has no support in browser.

Reply:
Then maybe we want to do a conditional import here? Something like this:
```ts 
const colorette = await isBrowser ? ... : import('colorette')
```

Reply:
we cannot do it on top level because dynamic import returns `Promise` and we cannot `await` it in `commonjs` module system

Reply:
> Can we safely import colorette in browser?

Yes, we can
Bu we cannot safely use it